docs/MO_DG/IR_and_opsets.md

```diff
@@ -30,15 +28,15 @@ A set of various operations used in a network is usually fixed for each deep lea
 It determines expressiveness and level of representation available in that framework.
 It may happen that a network that can be represented in one framework is hard or impossible to be represented in another one or should use significantly different graph because operation sets used in those two frameworks do not match.
```

Comment:
```suggestion
A network that can be represented in one framework is sometimes hard or impossible to be represented in another one or should use significantly different graph because operation sets used in those two frameworks do not match.
```